data/mods/gen7/random-teams.ts

```diff
@@ -116,6 +116,9 @@ export class RandomGen7Teams extends RandomTeams {
 				case 'icebeam':
 					if (hasAbility['Tinted Lens'] && !!counter['Status']) rejected = true;
 					break;
+				case 'morningsun':
+					if (counter.damagingMoves.length < 1) rejected = true;
+					break;
```

Comment:
What is this for?

Reply:
stealth rock morning sun dual screens solrock exists in gen 7